tests/plugins/colordialog/colordialog.js

```diff
@@ -0,0 +1,121 @@
+/* bender-tags: editor */
+/* bender-ckeditor-plugins: colordialog,wysiwygarea */
+
+( function() {
+	'use strict';
+
+	bender.editor = {};
+
+	var test = {
```

Comment:
There is no need to assign it to a variable, you can just write it as`bender.test( { ... } )`.